cloud/azure/storage/variables.tf

```
@@ -0,0 +1,2617 @@
+  type        = list
+}
+
+variable "prefixes" {
+  description = "Prefixes list to prepend between brackets on every monitors names before environment"
+  type        = list
+  default     = []
+}
+
+variable "filter_custom_includes" {
+  description = "List of tags to include when custom filtering is used"
+  type        = list
+  default     = []
+}
+
+variable "filter_custom_excludes" {
+  description = "List of tags to exclude when custom filtering is used"
+  type        = list
+  default     = []
+}
+
+variable "detectors_disabled" {
+  description = "Disable all detectors in this module"
+  type        = bool
+  default     = false
+}
+
+# Azure storage detectors specific
+
+variable "heartbeat_disabled" {
+  description = "Disable all alerting rules for heartbeat detector"
+  type        = bool
+  default     = null
+}
+
+variable "heartbeat_notifications" {
+  description = "Notification recipients list for every alerting rules of heartbeat detector"
+  type        = list
+  default     = []
+}
+
+variable "heartbeat_timeframe" {
+  description = "Timeframe for system not reporting detector (i.e. \"10m\")"
+  type        = string
+  default     = "20m"
+}
+
+# Blobservices_requests_error detectors
+
+variable "blobservices_requests_error_disabled" {
+  description = "Disable all alerting rules for blobservices_requests_error detector"
+  type        = bool
+  default     = null
+}
+
+variable "blobservices_requests_error_disabled_critical" {
+  description = "Disable critical alerting rule for blobservices_requests_error detector"
+  type        = bool
+  default     = null
+}
+
+variable "blobservices_requests_error_disabled_warning" {
+  description = "Disable warning alerting rule for blobservices_requests_error detector"
+  type        = bool
+  default     = null
+}
+
+variable "blobservices_requests_error_notifications" {
+  description = "Notification recipients list for every alerting rules of blobservices_requests_error detector"
+  type        = list
+  default     = []
+}
+
+variable "blobservices_requests_error_notifications_warning" {
+  description = "Notification recipients list for warning alerting rule of blobservices_requests_error detector"
+  type        = list
+  default     = []
+}
+
+variable "blobservices_requests_error_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of blobservices_requests_error detector"
+  type        = list
+  default     = []
+}
+
+variable "blobservices_requests_error_aggregation_function" {
+  description = "Aggregation function and group by for blobservices_requests_error detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".sum(by=['azure_resource_name', 'azure_resource_group_name'])"
+}
+
+variable "blobservices_requests_error_transformation_function" {
+  description = "Transformation function for blobservices_requests_error detector (mean, min, max)"
+  type        = string
+  default     = "max"
```

```suggestion
  default     = "min"
```
This transformation was inverted in previous implementation.